decidim-core/app/helpers/decidim/application_helper.rb

```diff
@@ -3,8 +3,6 @@
 module Decidim
   # Main module to add application-wide helpers.
   module ApplicationHelper
-    include Decidim::MetaTagsHelper
```

Comment:
Because I saw this was actually being included separately everywhere and including helpers in this way usually ends up creating autoloading/reloading issues.